src/app/api/utils/auth.tsx

```diff
@@ -129,13 +129,15 @@ export const authOptions: AuthOptions = {
               account.access_token,
               account.refresh_token,
               account.expires_at ?? 0,
-              JSON.stringify(profile),
+              profile,
```

Comment:
I checked, and both calls to `updateFxAData` called `JSON.stringify` on this argument, and then the function itself calls `JSON.parse` again. So I removed both and just passed the full object directly.

(This stringifying and parsing [predates the Next.js codebase by a long time](https://github.com/mozilla/blurts-server/blob/467a892c0b65fbc8889265054a266bd7f8d25433/src/db/tables/subscribers.js#L65), so there probably was a need for it back then.